recipes/lomap2/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "lomap2" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenFreeEnergy/Lomap/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 44f3935edd20937d152e0ecca54949cffcf74b80167de0a6a4f5ca6c9d0a5db6
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - "lomap = lomap.dbmol:startup"
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - networkx
+    - numpy
+    - python >=3.6
+    - rdkit
+
+test:
+  imports:
+    - lomap
+  requires:
+    - pip
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - lomap --help
```
Just to check the entry points.